runtime/dart_init.cc

```diff
@@ -253,6 +255,16 @@ Dart_Isolate ServiceIsolateCreateCallback(const char* script_uri,
 #endif  // FLUTTER_RUNTIME_MODE
 }
 
+static bool GetAssetAsBuffer(
+    const std::string& name,
+    std::vector<uint8_t>* data,
+    std::unique_ptr<DirectoryAssetBundle>& directory_asset_bundle,
+    fxl::RefPtr<ZipAssetStore>& asset_store) {
+  return (directory_asset_bundle &&
```

Comment:
Can we maybe split this up into multiple lines with early returns on failure conditions for readability?

Reply:
Oops, I didn't realize this was copied directly from the engine. Our preferences don't match but I don't have a strong opinion about this. Lets keep it as it.